apps/src/studio/Item.js

```diff
@@ -262,6 +280,9 @@ Item.prototype.update = function () {
       this.dir = Direction.NONE;
     }
   }
+
+  // Update this because animation speed may change as we alter direction:
+  this.animation_.setAnimationFrameDuration(this.getAnimationFrameDuration());
```

Comment:
If this always needs to happen when we set the direction, should we extract a `setDirection()` helper that ensures this?